client/modules/IDE/actions/project.js

```diff
@@ -72,7 +73,7 @@ export function clearPersistedState() {
 
 export function saveProject(autosave = false) {
   return (dispatch, getState) => {
-    const state = getState();
+    let state = getState();
```

Comment:
could you leave this as const, and create a new variable in the response? i think it will be clearer 😄